classes/trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9]
```

Comment:
- [ ] It will be great to have this implementation as a condition. For example if the ` @nameable.correct_name.length > 10` then you trim if not you just return the `@nameable.correct_name` without trimming because the name does not have characters more than 10 as per the requirement. Example ⬇️ 
`@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`

Reply:
Thankyou